frontend-react/src/components/Admin/AdminLastMileFailuresTable.tsx

```diff
@@ -9,24 +9,251 @@ import {
+            const parts = eachRow.receiver.split(".") || eachRow.receiver;
+            const org = parts[0] || "";
+            const recvrName = parts.slice(1).join(".");
+            const linkRecvSettings = `/admin/orgreceiversettings/org/${org}/receiver/${recvrName}/action/edit`;
+            const resends = fiterResends(eachRow.reportId);
+            const dataForDialog: DataForDialog = {
+                info: eachRow,
+                resends: resends,
+            };
+            return (
+                <tr key={`lastmile_row_${eachRow.pk()}`}>
+                    <td>{formatDate(eachRow.failedAt)}</td>
+                    <td>
+                        <Button
+                            type="button"
+                            unstyled
+                            className={"font-mono-xs"}
+                            title={"Show Info"}
+                            key={`details_${eachRow.pk()}`}
+                            onClick={() =>
+                                props.handleShowDetailsClick(
+                                    JSON.stringify(dataForDialog, null, 4)
+                                )
+                            }
+                        >
+                            {eachRow.reportId}
+                            {" ⧉"}
+                        </Button>
+                        <span
+                            className={"rs-resendmarker"}
+                            title={"Resends attempted."}
+                        >
+                            {resends.length ? " ⚠️ " : null}
+                        </span>
+                    </td>
+                    <td>
+                        <Link
+                            title={"Jump to Settings"}
+                            to={linkRecvSettings}
+                            key={`recv_link_${eachRow.pk()}`}
+                            className={"font-mono-xs"}
+                        >
+                            {eachRow.receiver}
+                        </Link>
+                    </td>
+                    <td>
+                        <Button
+                            key={`retry_${eachRow.pk()}`}
+                            onClick={() =>
+                                props.handleRetrySendClick(
+                                    JSON.stringify(eachRow, null, 2)
+                                )
+                            }
+                            type="button"
+                            size="small"
+                            className="padding-1 usa-button--outline"
+                            title="Requeue items for resend"
+                        >
+                            Resend...
+                        </Button>
+                    </td>
+                </tr>
+            );
+        });
+
+    return (
+        <Table
+            key="lastmiletable"
+            aria-label="List of failed sends"
+            striped
+            fullWidth
+        >
+            <thead>
```

Comment:
Can this component be refactored with the RS `Table` component? It seems like a lot of this JSX can get cleaned up.

Reply:
Humm.. how did we end up with a component named `<Table>` since there's _already_ one in Trussworks? It is super confusing. https://trussworks.github.io/react-uswds/?path=/story/components-table--striped